src/index.js

```diff
@@ -1,9 +1,13 @@
 import './style.css';
-// eslint-disable-next-line import/no-named-default
```

Comment:
- [ ] avoid disabling eslint rules as these rules help you write better code and hence making you a better developer